project_singlefamilydetached/singlefamilydetached_with_upgrades.yml

```diff
@@ -0,0 +1,1165 @@
+          - value: 3100
+            multiplier: Fixed (1)
+        lifetime: 24
+      - option: HVAC System Heating Fuel Oil|Oil Boiler, 80% AFUE
+        apply_logic:
+          - HVAC System Heating Fuel Oil|Oil Boiler, 72% AFUE
+          - HVAC System Heating Fuel Oil|Oil Boiler, 76% AFUE
+        costs:
+          - value: 16
+            multiplier: Size, Heating System (kBtu/h)
+          - value: 3100
+            multiplier: Fixed (1)
+        lifetime: 24                           
+  - upgrade_name: Ductless heat pump (displace electric baseboard today)
+    options: 
+      - option: HVAC System Heat Pump|MSHP, SEER 29.3, 14 HSPF
+        apply_logic: HVAC System Heating Electricity|Electric Baseboard
+        costs:
+          - value: 2510
+            multiplier: Fixed (1)
+          - value: 95
+            multiplier: Size, Heating System (kBtu/h)
+        lifetime: 15
+  - upgrade_name: Ductless Heat Pump (replace gas/propane/oil boiler at wear out)
+    options: 
+      - option: HVAC System Heat Pump|MSHP, SEER 29.3, 14 HSPF
+        apply_logic:
+          or:
+            - HVAC System Heating Natural Gas|Gas Boiler, 72% AFUE
+            - HVAC System Heating Natural Gas|Gas Boiler, 76% AFUE
+            - HVAC System Heating Natural Gas|Gas Boiler, 80% AFUE
+            - HVAC System Heating Natural Gas|Gas Boiler, 85% AFUE
+            - HVAC System Heating Natural Gas|Gas Boiler, 96% AFUE
+        costs:
+          - value: 2510
+            multiplier: Fixed (1)
+          - value: 95
+            multiplier: Size, Heating System (kBtu/h)
+        lifetime: 15
+        reference_scenario: Reference Boiler (gas, propane, oil)
+      - option: HVAC System Heat Pump|MSHP, SEER 29.3, 14 HSPF
+        apply_logic:
+          or:            
+            - HVAC System Heating Propane|Propane Boiler, 72% AFUE
+            - HVAC System Heating Propane|Propane Boiler, 76% AFUE
+            - HVAC System Heating Propane|Propane Boiler, 80% AFUE
+            - HVAC System Heating Propane|Propane Boiler, 96% AFUE
+        costs:
+          - value: 2510
+            multiplier: Fixed (1)
+          - value: 95
+            multiplier: Size, Heating System (kBtu/h)
+        lifetime: 15
+        reference_scenario: Reference Boiler (gas, propane, oil)
+      - option: HVAC System Heat Pump|MSHP, SEER 29.3, 14 HSPF
+        apply_logic:
+          or: 
+            - HVAC System Heating Fuel Oil|Oil Boiler, 72% AFUE
+            - HVAC System Heating Fuel Oil|Oil Boiler, 76% AFUE
+            - HVAC System Heating Fuel Oil|Oil Boiler, 80% AFUE
+            - HVAC System Heating Fuel Oil|Oil Boiler, 90% AFUE
+            - HVAC System Heating Fuel Oil|Oil Boiler, 95% AFUE
+        costs:
+          - value: 2510
+            multiplier: Fixed (1)
+          - value: 95
+            multiplier: Size, Heating System (kBtu/h)
+        lifetime: 15
+        reference_scenario: Reference Boiler (gas, propane, oil)
+
+##Thermostat
```

Comment:
I have some thermostat adjustment upgrades if you want to include them in this file

Reply:
Not sure. @ekpresent ?